gridt/controllers/follower.py

```diff
@@ -99,7 +95,6 @@ def get_leaders(user: User, movement: Movement, session: Session) -> list:
         .filter(
             UserToUserLink.follower_id == user.id,
             UserToUserLink.movement_id == movement.id,
-            not_(UserToUserLink.leader_id.is_(None)),
             UserToUserLink.destroyed.is_(None)
```

Comment:
Is this not still working with null links?

Reply:
No, line 102 is getting removed in this pull request because it isn't needed now that we aren't inserting null links in the system.